modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/singlecellsimulationtrace/v0.1.3.json

```diff
@@ -0,0 +1,41 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/simulation/singlecellsimulationtrace/v0.1.2/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:SingleCellSimulationTraceShape",
+      "@type": "sh:NodeShape",
+      "targetClass": "nsg:SingleCellSimulationTrace",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0/shapes/EntityShape"
+        },
+        {
+          "property": [
+            {
+              "path": "nsg:preview",
```

Comment:
I suggest to use schema:image for the preview. This is how preview is represented elsewhere.